tools/please_pex/pex_main.py

```diff
@@ -136,6 +136,19 @@ def load_module(self, fullname):
         sys.modules[fullname] = module
         return module
 
+    def get_code(self, fullname):
+        from pkgutil import read_code
+
+        module_path = fullname.replace('.', '/')
+        if zipfile.is_zipfile(sys.argv[0]):
+            zf = zipfile.ZipFile(sys.argv[0])
+            for name in zf.namelist():
+                if name.startswith(module_path):
+                    with open(name, 'rb') as f:
```

Comment:
not sure I follow the logic here... it walks through the whole zipfile and imports the first thing it finds that has a prefix of the thing they asked for?

I'm not sure we should do any zipfile stuff in here at all, can we just delegate to the builtin implementation?